recipes/assimp/4.1.0/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
```

Comment:
CCI does not allow more then 1 version per PR. You will need to create separated PRs.